lib/plugins/TaskRunner/LAMPApp.js

```diff
@@ -133,22 +138,38 @@ class LAMPApp extends Script {
     }
   }
 
-  // a meta-script that installs all apache/php-related scripts
-  addScriptApachePhp() {
+  // a meta-script that installs all apache/mysql/php related scripts.
+  addScriptApacheMysqlPhp() {
+    this.addScriptMysqlCnfOptions();
     this.addScriptPhpConstants();
     this.addScriptPhpIniOptions();
     this.addScriptApacheMods();
     this.addScriptPhpMods();
     this.addScriptRestartApache();
+    this.addScriptRestartMysql();
   }
 
-  // a meta-script that runs all setup commands
+  // a meta-script that runs all setup commands.
   addScriptSetup() {
     this.addScriptHeader();
     this.addScriptCliDefines();
     this.addScriptInstallPackages();
   }
 
+  addScriptMysqlCnfOptions() {
+    MYSQLCNF_PATH="/etc/mysql/conf.d"
```

Comment:
This constant should be moved to the top of the file.

Reply:
This function should also be near the addScriptRestartMysql file.

Reply:
I've got the mysql functions in a separate function and moved the constant to the constants.js file with the rest of the constants.